cibuild.sh

```diff
@@ -72,14 +72,12 @@ restore_nuget()
 
     rm $package_name 2>/dev/null
     curl -O https://dotnetci.blob.core.windows.net/roslyn/$package_name
-    unzip -foq $package_name -d ~/
+    unzip -uoq $package_name -d ~/
     if [ $? -ne 0 ]; then
         echo "Unable to download NuGet packages"
         exit 1
     fi
 
-    rm $package_name 2>/dev/null
```

Comment:
Up a couple lines we already check that to see if we should restore. I think this `rm` was just extra and me being dumb.